portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/UserManagement.testcase

```diff
@@ -38,12 +38,61 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8143 | Test Summary: Owner can delete users"
-	@ignore = "true"
 	@priority = "5"
 	test ViewOwnerCanDeleteUser {
+		ACNavigation.goToSettings();
+
+		ACSettings.goToUserManagement();
+
+		ACSettings.addUser(userInvite = "user@liferay.com");
+
+		ACUtils.searchBar(searchItem = "user");
+
+		ACSettings.assertUserListEmail(userEmail = "user@liferay.com");
+
+		ACSettings.deleteUser(userDelete = "user@liferay.com");
```

Comment:
Hi @joshitagaki. What do you think if after deleting this user, you create another user to check if a user created with one account can be deleted by a different account?

Reply:
I'll move the `ACSettings.addUser(userInvite = "user2@liferay.com");` to the first user and have `michelle.hoshi@faro.io` attempt to delete

Reply:
Yes, that's a good idea! Thanks!